introspection.go

```diff
@@ -266,6 +266,14 @@ mutation operations.`,
 					return nil, nil
 				},
 			},
+			"subscriptionType": &Field{
+				Description: `If this server support subscription, the type that ' +
+                   'subscription operations will be rooted at.`,
+				Type: __Type,
+				Resolve: func(p ResolveParams) (interface{}, error) {
+					return nil, nil
+				},
+			},
```

Comment:
@pyros2097 there was a [PR](https://github.com/graphql-go/graphql/pull/87) for this and I commented quote:
> If we add a subscriptionType, we will allow users to actually write valid queries that contains subscription keyword on it, since we did not implemented the subscription feature yet, it might be a little confusing for users I think.